geopolars/src/geoseries.rs

```diff
@@ -5,17 +5,79 @@ use arrow2::array::{
     ArrayRef, BinaryArray, BooleanArray, MutableBinaryArray, MutableBooleanArray,
     MutablePrimitiveArray, PrimitiveArray,
 };
-use geo::{Geometry, Point};
+use geo::algorithm::affine_ops::AffineTransform;
+use geo::{map_coords::MapCoords, Geometry, Point};
 use geozero::{CoordDimensions, ToWkb};
-use polars::prelude::{PolarsError, Result, Series};
+use polars::prelude::{PolarsError, Result, Series, TakeRandom};
+use std::convert::Into;
 
 pub enum GeodesicLengthMethod {
     Haversine,
     Geodesic,
     Vincenty,
 }
 
+/// Used to express the origin for a given transform. Can be specified either be with reference to
```

Comment:
Seems like a decent amount of this file's diff is out of date with `master` as well. I think it'll be easier to review with the correct diff

Reply:
Should be updated now. Don't need a full review here, just wanted to sketch this out as a possible solution to braodcasting / accepting either a value or a Series for series transforms. 

If we like it, I can go back and implement on the other functions where it makes sense.